packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -45,8 +45,8 @@ const AppContents = observer(({ children }) => {
 
     React.useEffect(() => {
         if (is_logged_in && user_id) {
-            RudderStack.setAccountType(loginid.substring(0, 2));
-            RudderStack.identifyEvent(user_id, {
+            Analytics.setAccountType(loginid.substring(0, 2));
```

Comment:
and account_type can be set once with fn I mentioned

Reply:
that is settiung the attributes on growthbook instance only

Reply:
now using `setAttributes` everywhere to modify `account_type` or `user_id` ..etc